db.json

```diff
@@ -3961,14 +3961,14 @@
         "kwargs": {
             "name": "{{name}}"
         },
-        "last_edit": "Wed Jun 24 19:23:43 2020",
+        "last_edit": "Sat Jun 27 23:04:54 2020",
         "lightpath": false,
         "location_group": "None",
         "name": "icl_sds_sel_b",
         "prefix": "ICL:SDS:SEL:B",
         "stand": null,
         "type": "pcdsdevices.happi.containers.LCLSItem",
-        "z": -1.0
+        "z": 999999.0
```

Comment:
We should clarify the `happi` documention to say that `-1` is the null value here. This value change is seen as a valid "wow we have a very far away device on the beamline but it shouldn't be on the lightpath because `lightpath==False`"

Reply:
Verifying `0 <= z < 4000` may not be a bad idea, either?

I have some vague recollection that some GUNB stuff is actually just below Z=0, but `Z>=0` is probably good enough regardless.

Reply:
Restricting the range of `z` also makes it clear what the usage should be